_posts/2020-03-02-frontend-developer-love-2020.md

```diff
@@ -0,0 +1,500 @@
+---
+layout: post
+authors: [elke_heymans]
+title: "Frontend Developer Love 2020"
+image: /img/frontend-developer-love-2020.png
+tags: [Conference, Vue.js, React, GraphQL, WebAssembly, a11y]
+category: Conference
+comments: true
+---
+
+# Table of Contents
+
+* [Introduction](#introduction)
+* [How thinking small is changing software development big time](#how-thinking-small-is-changing-software-development-big-time-by-sander-hoogendoorn)
+* [How to pack your Webpack](#how-to-pack-your-webpack-by-johannes-ewald)
+* [Svelte: the last framework we need?](#svelte-the-last-framework-we-need-by-alexander-esselink)
+* [Serverless gives you wings](#serverless-gives-you-wings-by-yan-cui)
+* [Modern solutions for e2e testing](#modern-solutions-for-e2e-testing-by-anastasiia-dragich)
+* [Practical a11y for web apps](#practical-a11y-for-web-apps-by-bob-bijvoet)
+* [Micro-interactions with React Spring](#micro-interactions-with-react-spring-by-emma-bostian)
+* [But, you're not Facebook](#but-youre-not-facebook-by-kitze)
+* [Beats, rhymes and unit tests](#beats-rhymes-and-unit-tests-by-tony-edwards)
+* [GraphQL without a database](#graphql-without-a-database-by-roy-derks)
+* [DX is the new black. Learnings from using Nuxt and Storybook at scale](#dx-is-the-new-black-learnings-from-using-nuxt-and-storybook-at-scale-by-aurélie-violette)
+* [Refactor your life](#refactor-your-life-by-noer-paanakker--sima-milli)
+* [Blazor with WebAssembly](#blazor-with-webassembly-by-don-wibier)
+* [Audio Streaming - Using WebRTC for building your own Voice AIs](#audio-streaming---using-webrtc-for-building-your-own-voice-ais-by-lee-boonstra)
+* [The future of real-time, offline, data](#the-future-of-real-time-offline-data-by-nader-dabit)
+* [The state of WebAssembly](#the-state-of-webassembly-by-sendil-kumarn)
+* [Conclusion](#conclusion)
+
+# Introduction
+
+The Frontend Developer Love conference day was the first of 3 conference days that I was visiting in Amsterdam.
+While the other 2 days were focussed on VueJS, the first day was a day filled with more generic frontend topics.
```

Comment:
Het woord "day" komt nogal veel voor en dat maakt het moeilijker om te lezen. Misschien is het beter om deze zin te wijzigen naar: 
```suggestion
While the last 2 days were focussed on VueJS, the first day was filled with more generic frontend topics.
```